tests/modules/misc/fontconfig/multiple-font-packages.nix

```diff
@@ -1,15 +1,15 @@
-{ pkgs, ... }:
+{ realPkgs, ... }:
+
 {
-  config = {
-    home.packages = [
-      pkgs.comic-relief
-      pkgs.unifont
-    ];
+  fonts.fontconfig.enable = true;
+  home.packages = [
+    realPkgs.comic-relief
+    realPkgs.unifont
```

Comment:
I'd prefer to not introduce dependency on `realPkgs` for random packages, if we can.